docs/platform/release-notes/tizen-9-0-m2.md

```diff
@@ -0,0 +1,663 @@
+  - Tizen.Multimedia.Remoting
+    - WebRTCStatisticsProperty enum have been updated.
+    - Deprecated APIs of MediaController server has been removed.
+  - Tizen.Multimedia.Vision
+    - New QR APIs have been added.
+  - Tizen.Content.Download
+    - New caching APIs have been added.
+  - Tizen.Location.Geofence has been deprecated.
+  - Tizen.Messaging.Email has been deprecated.
+  - Tizen.Messaging.Messages has been deprecated.
+  - Tizen.Applications.ComponentBased
+    - TimeZoneChanged event handler has been added.
+  - Tizen.Applications.Common
+    - TimeZoneEvent has been added.
+  - Tizen.Applications.AttachPanel has been deprecated.
+  - Tizen.Multimedia.Vision
+    - InferenceTargetType and InferenceBackendType have been removed.
+  - Tizen.Multimedia.Util
+    - Deprecated APIs has been removed.
+  - Tizen.Multimedia.StreamRecorder has been removed.
+  - Tizen.Uix.Tts
+    - New TTS synthesized PCM APIs have been added.
+  - Tizen.Uix.InputMethod
+    - RotationChanged event has been deprecated.
+    - Some APIs depending on ElmSharp has been removed.
+  - Tizen.Account.FidoClient has been deprecated.
+  - Tizen.NUI.Scene3D
+    - MotionData class has been added.
+  - Tizen.NUI
+    - Scrollable and ScrollViewEffect has been deprecated.
+    - WeakEvent has been added.
+  - Tizen.Core has been added.
+    - New main loop model to provide improved functionality has been added.
+  - Tizen.Network.Connection
+    - WPA3 security type has been added.
+  - TizenWebView has been removed.
+  - Tizen.NUI.Wearable has been deprecated.
+
+
+### Machine learning
+
+## New and changed features
+
+### Machine Learning (ML) API updates
+  -  ML.Service API Updates
+    - ML Service API supports application developers to use AI offloading service using various protocols such as TCP, MQTT, and other custom connection via NNStreamer-Edge.
+  - Add NNFW enum type to support TensorRT and QNN.
+
+### NNStreamer and Related modules update
+  - NNStreamer has been upgraded to version 2.4.2.
+    - Add new inference filter to support the ONNX Runtime.
+    - Add new inference filter to support the SNPE version 2 (Qualcomm Neural Processing SDK).
+    - Add new inference filter to support the QNN (Qualcomm AI Engine Direct).
+    - Add new inference filter to support the TensorRT version 10.
+    - Add new inference filter to support the ncnn engine (Tencent).
+    - Add new inference filter to support the ExecuTorch (Experimental).
+
+  - NNStreamer-Edge has been upgraded to version 0.2.6.
+    - Support custom connection (user defined protocol).
+    - Support Multi-Connectivity Framework (MCF) using custom connection.
+
+  - MLOps-Agent has been upgraded to version 1.8.6.
+    - Parse the path for AI model or recource files from installed RPK.
+
+  - Support ML computation offloading to an Android device (Experimental).
+
+## Fixes
+  - Reported bugs in NNStreamer and ML API have been fixed.
+
+## Known issues
```

Comment:
```suggestion
#### Known issues
```

Reply:
Please confirm if this is okay. I have considered that the should "Known issues" fall under the "### Machine Learning" (third-level heading) category. @seokwon-jung